src/containers/email/types.ts

```diff
@@ -0,0 +1,28 @@
+import { Neighborhoods } from '../../assets/content';
+
+export enum EmailType {
+  INACTIVE = 'Inactive',
+  NEIGHBORHOOD = 'Neighborhood',
+}
+
+export interface EmailerFilters {
+  activityCountMin: number;
```

Comment:
❓Should `activityCountMin` also be optional (i.e. `activityCountMin?: number;`)?

Reply:
I think since there's a hard lower limit on activity counts at 0, making it an optional prop would just introduce unnecessary complexity - if `activityCountMin` is 0, then that basically means there's no minimum anyways